Draft for introduction (#8)
Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper: paper.tex

\end{abstract}

\section{Introduction}

> The development of generative Machine Learning models made huge steps forward in the last years, considering the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image generating models like Vision Transformer (ViT) models already created fake images being reported as real by media. Other news showed artists winning art competitions with generated art. This creates an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.

Reviewer: ```suggestion
The development of generative Machine Learning models made huge steps forward in the last years, considering the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image generating models like Vision Transformer (ViT) already created fake images being reported as real by media. Other news showed artists winning art competitions with generated art. This creates an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.
```

> The development of generative Machine Learning models make huge steps forward in the last years considering comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. There were already scenarios were fake images, created by image-generation models such as Vision Transformer (ViT) models, were reported as real ones by media. Other news showed artists winning art competitions with generated art. That means that there is an interest in detecting, which texts or images are generated and which ones are real pictures or human-written texts.

Reviewer 2: ```suggestion
The development of generative Machine Learning models made huge steps forward in the last years, considering the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image generating models like Vision Transformer (ViT) models already created fake images being reported as real by media. Other news showed artists winning art competitions with generated art. This creates an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.
```

> Therefore researchers evaluated different approaches to detect fake images and texts. Fortunately some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.

Reviewer: ```suggestion
Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images. This raises the question, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```

> Therefore researchers evaluated different approaches to detect fake images and texts. But these attempts are from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.

Reviewer 2: ```suggestion
Therefore researchers evaluated different approaches to detect fake images and texts. Fortunately some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```

> It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, existing detectors need to be improved or better detectors need to be developed that are robust against these techniques. But this preconditions a good understanding of the way attacks are performed.

Reviewer 3: i dont like "exist, existing"

> It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, existing detectors need to be improved or better detectors need to be developed that are robust against these techniques. But this preconditions a good understanding of the way attacks are performed.

Reviewer: ```suggestion
It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, existing detectors need to be improved or better detectors need to be developed that are robust against these techniques. But this requires a good understanding how the attacks are performed.
```

> It is necessary to evaluate whether or not such techniques exist. If not one could rely on detectors, which can easily be easily tricked. If there are techniques exist that allow easy and efficient manipulation, better detectors, which are robust against those, need to be developed. Another solution would be to improve the robustness and accuracy of existing detectors, when understanding how the attacks can be crafted.

Reviewer 2: ```suggestion
It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, existing detectors need to be improved or better detectors need to be developed that are robust against these techniques. But this preconditions a good understanding of the way attacks are performed.
```

Therefore we came up with the following research questions:

RQ1: How can generated images be manipulated to trick fake image detectors?

RQ2: How can generated texts be manipulated to trick fake text generators?

> The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude the work.

Reviewer: ```suggestion
The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude the paper.
```

Reviewer 3: maybe "this paper"

> The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and fake text detectors. Chapter 3 presents the methodology of our work, followed by chapter 4 which shows the results of our experiments. Afterwards in  Chapter 5 those results will be discussed and our work will be concluded in chapter 6.

Reviewer 2: ```suggestion
The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude the work.
```

\end{abstract}

> \section{Introdution}

Reviewer 3: ```suggestion
\section{Introduction}
```

> The development of generative Machine Learning models made huge steps forward, when it comes to comparison with human crafted artefacts. Large Language Models enable the generation of texts in all kind of different flavours and styles. Image generating models like ViT models can generate images, that already was used create fake images, that were reported as true by media. Other news showed artists winning art competitions with generated art. So there is an interest to detect, which texts or images are generated and which ones are real pictures or human-written text.

Reviewer: ```suggestion
The development of generative Machine Learning models made huge steps forward, when it comes to comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image-Generation models like Vision Transformer(ViT) can generate images. Some of these fake images were already reported as true by the media. Other news showed artists winning art competitions with generated art. So there is an interest to detect, which texts or images are generated and which ones are real images or human-written texts.
```

Reviewer 2: Some good changes, though I won't change generating into generation, since as far as I know, the right word is generating

Reviewer: naja soll es heißen "bild generierungs modelle wie..." oder "bild generations modelle wie..." oder? abhängig davon is es ing oder on aber idk haha, ihr könnt das gern entscheiden.

Reviewer 2: Naja, ne Generierung erzeugt ein Bild meiner Meinung nach, Generations ist ja eher dann sowas wie Genration Z beispielsweise xD

> The development of generative Machine Learning models made huge steps forward, when it comes to comparison with human crafted artefacts. Large Language Models enable the generation of texts in all kind of different flavours and styles. Image generating models like ViT models can generate images, that already was used create fake images, that were reported as true by media. Other news showed artists winning art competitions with generated art. So there is an interest to detect, which texts or images are generated and which ones are real pictures or human-written text.

Reviewer 3: ```suggestion
The development of generative Machine Learning models made huge steps forward in the last years, when it comes to the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. There were already scenarios were fake images, created by image-generation models such as Vision Transformer (ViT) models, were reported as real ones by media. Other news showed artists winning art competitions with generated art. That means that there is an interest in detecting, which texts or images are generated and which ones are real pictures or human-written texts.
```

> The development of generative Machine Learning models made huge steps forward, when it comes to comparison with human crafted artefacts. Large Language Models enable the generation of texts in all kind of different flavours and styles. Image generating models like ViT models can generate images, that already was used create fake images, that were reported as true by media. Other news showed artists winning art competitions with generated art. So there is an interest to detect, which texts or images are generated and which ones are real pictures or human-written text.

Reviewer 2: ```suggestion
The development of generative Machine Learning models make huge steps forward considering comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kind of different flavours and styles. Image generating models like Vision Transformers models (ViT) can generate images, that already tricked media. Other news showed artists winning art competitions with generated art. So there is an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.
```

> The development of generative Machine Learning models made huge steps forward, when it comes to comparison with human crafted artefacts. Large Language Models enable the generation of texts in all kind of different flavours and styles. Image generating models like ViT models can generate images, that already was used create fake images, that were reported as true by media. Other news showed artists winning art competitions with generated art. So there is an interest to detect, which texts or images are generated and which ones are real pictures or human-written text.

Reviewer 3: ```suggestion
The development of generative Machine Learning models made huge steps forward in the last years, considering the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image generating models like Vision Transformer (ViT) models already created fake images being reported as real by media. Other news showed artists winning art competitions with generated art. This creates an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.
```

> So researchers evaluated different approaches to detect fake images and texts. But from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artifacts (e.g. images and texts) to reduce the accuracy of detectors.

Reviewer: ```suggestion
Therefore researchers evaluated different approaches to detect fake images and texts. But from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artifacts (e.g. images and texts) to reduce the accuracy of detectors.
```

> So researchers evaluated different approaches to detect fake images and texts. But from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artifacts (e.g. images and texts) to reduce the accuracy of detectors.

Reviewer 3: ```suggestion
Therefore researchers evaluated different approaches to detect fake images and texts. But these attempts are from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artifacts (e.g. images and texts) to reduce the accuracy of detectors.
```

> So researchers evaluated different approaches to detect fake images and texts. But from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artifacts (e.g. images and texts) to reduce the accuracy of detectors.

Reviewer 2: Would also change artifacts to artfecats to make it unique. (It's also artefacts in the above part)
```suggestion
Therefore researchers evaluated different approaches to detect fake images and texts. But from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```

> So researchers evaluated different approaches to detect fake images and texts. But from other computer vision tasks like object detection, adversarial attacks are known, that can reduce the performance by artificially manipulating real images to reduce the performance of e.g. visual object detection models. This raises the questions, whether and how it is possible to manipulate generated artifacts (e.g. images and texts) to reduce the accuracy of detectors.

Reviewer 3: ```suggestion
Therefore researchers evaluated different approaches to detect fake images and texts. Fortunately some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual obejct detection, by artificially manipulating real images. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```

> It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, better detectors need to be developed that are robust against these techniques. Also one might be able to improve the robustness and accuracy of existing detectors, when one understands how the attacks can be crafted.

Reviewer: ```suggestion
It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, better detectors need to be developed that are robust against these techniques. Also one might be able to improve the robustness and accuracy of existing detectors, when one understands how the attacks can be crafted.
```
little bit of a loop no?, dont know if i like it (good to know how to trick detectors -> more robust detectors -> good to know how to trick detectors)

Author: Agree it is like a loop, however that's a usual thing in security: research tries to find new ways to attack systems -> fix the attack surface in the system - > try again to attack the system. The goal is to be faster then the bad guys

> It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, better detectors need to be developed that are robust against these techniques. Also one might be able to improve the robustness and accuracy of existing detectors, when one understands how the attacks can be crafted.

Reviewer 3: ```suggestion
It is necessary to evaluate whether or not such techniques exist. If not one could rely on detectors, which can be easily tricked. If ther are techniques that allow easy and efficient manipulation, better detectors, which are robust against those, need to be developed. Another solution would be to improve the robustness and accuracy of existing detectors, when understanding how the attacks can be crafted.
```

> It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, better detectors need to be developed that are robust against these techniques. Also one might be able to improve the robustness and accuracy of existing detectors, when one understands how the attacks can be crafted.

Reviewer 2: I would then change it like this, to consider Levis comment:
```suggestion
It is necessary to evaluate whether such techniques exist, because otherwise one could rely on detectors, which can easily be tricked. If easy efficient manipulation techniques exist, existing detectors need to be improved or better detectors need to be developed that are robust against these techniques. But this preconditions a good understanding of the way attacks are performed.
```

Therefore we came up with the following research questions:

RQ1: How can generated images be manipulated to trick fake image detectors?

RQ2: How can generated texts be manipulated to trick fake text generators?

> The report is structured as followed: After this introduction chapter 2 presents related work, especially text and image generating models as well  fake image and fake text detectors. Chapter 3 presents the methodology of our work. Chapter 4 presents the results our experiments. Chapter 5 discusses the results and chapter 6 will conclude the work.

Reviewer: ```suggestion
The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generation models as well as fake image and fake text detectors. Chapter 3 presents the methodology of our work. Chapter 4 presents the results of our experiments. Chapter 5 discusses the results and chapter 6 will conclude the work.
```
is it ok to use numbers written like 1,2,3 or is one, two, three better?
also why 2 times results?

Reviewer 2: I wouldn't change follows and generating, It's correct in my opinion.

Reviewer 2: It's one time presenting results (results section) and one time discussing results (discussion section)

Reviewer: idk i googled it and they said follwos, but again u can decide. 
![grafik](https://github.com/Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper/assets/102458061/923b3cbc-9639-4256-b0bb-c32341be8517)

Reviewer 2: Yeah, you are right, I read it the other way round, thought you wanted to swap follows for followed xD

> The report is structured as followed: After this introduction chapter 2 presents related work, especially text and image generating models as well  fake image and fake text detectors. Chapter 3 presents the methodology of our work. Chapter 4 presents the results our experiments. Chapter 5 discusses the results and chapter 6 will conclude the work.

Reviewer 3: ```suggestion
The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generation models as well as fake image and text detectors. Chapter 3 presents the methodology of our work, followed by chapter 4 which shows the results our experiments. Afterwards in Chapter 5 those results will be discussed and our work will be concluded in chapter 6.
```

> The report is structured as followed: After this introduction chapter 2 presents related work, especially text and image generating models as well  fake image and fake text detectors. Chapter 3 presents the methodology of our work. Chapter 4 presents the results our experiments. Chapter 5 discusses the results and chapter 6 will conclude the work.

Reviewer 2: Would do it like this:
```suggestion
The report is structured as followed: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and text detectors. Chapter 3 presents the methodology of our work before chapter 4 presents the results our experiments. The results then are discussed in chapter 5 and chapter 6 will conclude the work.
```

> The report is structured as followed: After this introduction chapter 2 presents related work, especially text and image generating models as well  fake image and fake text detectors. Chapter 3 presents the methodology of our work. Chapter 4 presents the results our experiments. Chapter 5 discusses the results and chapter 6 will conclude the work.

Reviewer 3: ```suggestion
The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude the work.
```
